src/Furniture/Chair.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Vendor\FurnitureCalculator\Furniture;
+
+class Chair extends AbstractFurniture implements FurnitureInterface
+{
+    public function quantityOfMaterial()
+    {
+        // some operations with sizes
+        $quantity = 3;
```

Comment:
it allways return 3. If this method is to return the quality, it should have the name getQualityOfMaterial. And you need some variable (private property) for storing it